portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,7 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+    ['Hi', 'Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!', 'Greeting', "According to all known laws of aviation, there is no way a bee should be able to fly. Its wings are too small to get its fat little body off the ground. The bee, of course, flies anyway because bees don't care what humans think is impossible."];
```

Comment:
If the line is too long, you can optionally wrap into multiple lines to improve readability.